projects/packages/forms/src/contact-form/class-contact-form.php

```diff
@@ -1586,11 +1606,15 @@ public static function get_mail_content_type() {
 	 *
 	 * This helps to ensure correct parsing by clients, and also helps avoid triggering spam filtering rules
 	 *
+	 * @param string $title - title of the email.
+	 * @param string $response_link - the link to the response.
+	 * @param string $form_link - the link to the form.
 	 * @param string $body - the message body.
+	 * @param string $footer - the footer containing meta information.
 	 *
 	 * @return string
 	 */
-	public static function wrap_message_in_html_tags( $body ) {
+	public static function wrap_message_in_html_tags( $title, $response_link, $form_link, $body, $footer ) {
```

Comment:
I think it would be nice to add tests to cover all the changes we're making to that method.

Reply:
Done!